cpp/csp/engine/Struct.h

```diff
@@ -368,7 +369,7 @@ class ArrayStructField : public NonNativeStructField
 
     void deepcopyFrom( const Struct * src, Struct * dest ) const override
     {
-        deepcopy( value( src ), value( dest ) );
+        //deepcopy( value( src ), value( dest ) );
```

Comment:
is this intentional?

Reply:
Accientally commited this change along with the merge since i fixed a conflict here.
This PR is still WIP, not ready for review